assets/dev/scss/frontend/_container.scss

```diff
@@ -137,8 +137,8 @@
 		width: 100%;
 		max-width: var(--content-width);
 		margin: 0 auto;
-		padding-inline-start: 0;
-		padding-inline-end: 0;
```

Comment:
@mykytamurzin Does this not change default padding inline-start from 0px to 10px?

```
	--padding-block-start: var(--container-default-padding-block-start, 10px);
	--padding-inline-end: var(--container-default-padding-inline-end, 10px);
	--padding-block-end: var(--container-default-padding-block-end, 10px);
	--padding-inline-start: var(--container-default-padding-inline-start, 10px);